### ISSUE TITLE
image completion example

### ISSUE THREAD

**Issue by zhenzey** [CompVis/taming-transformers#161]
Thanks for the nice codes and tutorials. I was wondering whether you can provide an example for image completion using the VQGAN with python. Streamlit is relatively hard to use on my cluster. Thanks a lot!

**Comment by nicolasfischoeder** [comment]
Yeah I would also like to have something like that. Is that possible?
